src/App.js

```diff
@@ -1,49 +1,46 @@
 import React from 'react';
 import Header from './Header';
 import Footer from './Footer';
+import BestBooks from './BestBooks';
 import 'bootstrap/dist/css/bootstrap.min.css';
-import {
-  BrowserRouter as Router,
-  Switch,
-  Route
-} from "react-router-dom";
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
 
 class App extends React.Component {
-
   constructor(props) {
     super(props);
     this.state = {
       user: null,
-    }
+    };
   }
 
-  loginHandler = (user) => {
+  loginHandler = user => {
     this.setState({
       user,
-    })
-  }
+    });
+  };
 
   logoutHandler = () => {
     this.setState({
       user: null,
-    })
-  }
+    });
+  };
 
   render() {
     return (
       <>
         <Router>
           <Header user={this.state.user} onLogout={this.logoutHandler} />
           <Switch>
-            <Route exact path="/">
+            <Route exact path='/'>
               {/* TODO: if the user is logged in, render the `BestBooks` component, if they are not, render the `Login` component */}
```

Comment:
Here is a slight nudge:

You can use a ternary here if you'd like: this.state.user ? show <BestBooks/> : <Login /> (This is not the exact code of course but this will hopefully help when you get here 👍